src/domain/shared/index.types.d.ts

```diff
@@ -19,6 +23,11 @@ type DisplayAmount<T extends DisplayCurrency> = {
   amount: number
 }
 
+type DisplayCurrencyObject<T extends DisplayCurrency> = {
```

Comment:
New object type coming in here that's used in `DisplayPriceRatio`

Reply:
and if we only modify DisplayAmount?

Reply:
Yup, but this would be a transition step while we have the two separate types in our codebase still. I haven't changed any of the application code as yet to use this new type. Just establishing the pattern first and looking for feedback on that as our eventual target.